create-database.sql

```diff
@@ -51,3 +51,4 @@ INSERT INTO ProductInventory (ID,ProductID,WarehouseAmount,WarehouseID)
 	VALUES (1, 1, 2, 1);
 INSERT INTO ProductInventory (ID,ProductID,WarehouseAmount,WarehouseID)
 	VALUES (2, 1, 4242, 2);
+    
```

Comment:
You accidentally removed the EoF, the last line of the file should be empty.
![image](https://github.com/mate-academy/rda_task_5_stored_procedures/assets/87720583/0331920e-a52f-4786-b919-6a4f867743c1)